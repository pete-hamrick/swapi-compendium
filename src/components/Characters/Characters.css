.character {
  list-style: none;
  border: 5px solid gold;
  width: 300px;
  height: 300px;
  margin: 10px;
}
