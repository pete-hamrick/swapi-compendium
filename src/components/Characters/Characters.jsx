import React from 'react'

export default function Characters({ character }) {
  return <li>{character.name}</li>
}
