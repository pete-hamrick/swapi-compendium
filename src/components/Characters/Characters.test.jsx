import React from 'react'
import { render } from '@testing-library/react'
import Characters from './Characters'

it('should display the Character component', () => {
  const character = { name: 'Babu Frik', eye_color: 'yellow', height: '22', birth_year: '50BBY' }
  const { container } = render(<Characters character={character} />)

  expect(container).toMatchSnapshot()
})
