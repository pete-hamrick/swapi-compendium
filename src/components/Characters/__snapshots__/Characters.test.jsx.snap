// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the Character component 1`] = `
<div>
  <li
    class="character"
  >
    <h5>
      Babu Frik
    </h5>
    <p>
      Eye Color: 
      yellow
    </p>
    <p>
      Height: 
      22
      cm
    </p>
    <p>
      Birth Year: 
      50BBY
    </p>
  </li>
</div>
`;
